src/app/content/utils.ts

```diff
@@ -8,16 +8,16 @@ export { stripIdVersion } from './utils/idUtils';
 export { scrollTocSectionIntoView } from './utils/domUtils';
 
 export const getContentPageReferences = (content: string) =>
-  (content.match(/\/contents\/([a-z0-9\-]+(@[\d\.]+)?)(:([a-z0-9\-]+(@[\d\.]+)?))?/g) || [])
+  (content.match(/"\/contents\/([a-z0-9\-]+(@[\d\.]+)?)(:([a-z0-9\-]+(@[\d\.]+)?))?/g) || [])
```

Comment:
What prompted the need to include the `"` in the match? And if it's because other text contained `/contents/` should this be `href="`